### DISCUSSION TITLE
Failed to load schemas

### DISCUSSION THREAD

**Discussion by itsdrshadow** [supabase/supabase#18646] [Questions]
Failed to load schemas
Error: <html> <head><title>503 Service Temporarily Unavailable</title></head> <body> <center><h1>503 Service Temporarily Unavailable</h1></center> </body> </html>

Reload schemas

**Comment by teomikitenko** [comment]
same

**Reply by itsdrshadow** [reply]
Hey there,


Thanks for contacting Supabase support. We're looking into this issue.


You can monitor the status on our status page here:

https://status.supabase.com/


Best regards,

Tyler Fontaine

Supabase Support Engineer


Recently got email from them
